modules/math_libs.cmake

```diff
@@ -122,15 +122,18 @@ set(ATLAS_BLAS_LIBS   f77blas cblas atlas)
 set(ATLAS_LAPACK_LIBS atlas lapack)
 
 #-------------------------------------------------------------------------------
-# OPENBLAS (no LAPACK in OPENBLAS)
+# OPENBLAS (contains also LAPACK)
```

Comment:
Hi Rado,

here I made important change - the famous OpenBLAS library contains also the LAPACK routines (see also https://github.com/xianyi/OpenBLAS/issues/203) . This has been exploited before in DIRAC-native-Windows buildup script, see https://gitlab.com/dirac/dirac/blob/master/maintenance/cdash/cdash.WinS12-Miro.bat#L34 .